telecine/src/main/java/com/jakewharton/telecine/TelecineComponent.java

```diff
@@ -1,15 +1,18 @@
 package com.jakewharton.telecine;
 
+import android.app.Application;
+import dagger.BindsInstance;
 import dagger.Component;
+import dagger.android.AndroidInjectionModule;
 import javax.inject.Singleton;
 
-@Singleton
-@Component(modules = TelecineModule.class)
+@Singleton @Component(modules = { AndroidInjectionModule.class, TelecineModule.class })
```

Comment:
do you happen to know why this seemed to work fine without declaring `AndroidInjectionModule.class` in the modules?